.github/actions/firebase/action.yml

```diff
@@ -13,4 +16,4 @@ runs:
     - name: Copy build/deployment files
       shell: bash
       run: |
-        cp  ${{ github.action_path }}/files/* ${{ inputs.working-directory }}
+        cp  ${{ github.action_path }}/files/* ${{ inputs.working-directory + '/' + inputs.package-name }}
```

Comment:
will need to verify this doesn't mess up when no package-name is given

Reply:
If doesn't work, you can make a conditional step with or without package-name.